### ISSUE TITLE
Recovery key documentation outdated since NC 13?

### ISSUE THREAD

**Issue by tflidd** [nextcloud/documentation#1340]
https://help.nextcloud.com/t/recovery-key-field-missing/50590

Please clarify and correct the documentation if that is not correct any more.

**Comment by MorrisJobke** [comment]
cc @schiessle @rullzer

**Comment by schiessle** [comment]
yes, be default we use the master key. In this setup no recovery key exists. You can still switch to per-user keys, then the recovery key is available again. So yes, we should update the documentation.

I see two options:

1. Make it clear that this option doesn't exists by default but only in the special "per-user-key-setup" which the admin needs to enable via occ right from the beginning

2. remove it completely because it is highly recommend to stick to the master key

**Comment by hdbreaker** [comment]
Where can I find my Master Key? I do not find it at any place!
